stroom-app/src/integrationTest/java/stroom/connectors/KafkaExternalLoader.java

```diff
@@ -0,0 +1,75 @@
+package stroom.connectors;
```

Comment:
I used this integration tests to drive out the issues with the dynamic loading, without having to run up the entire application